### ISSUE TITLE
[2.0.1] New 'deprecated' recipes?

### ISSUE THREAD

**Issue by AlphaApotheosis** [GTNewHorizons/NewHorizons#2076]
These recipes were not present in 2.0.0. Is there a reason a 'deprecated' recipe was added to 2.0.1? Having these recipes available at all, after they were removed in 1.5-2.0, seems like it will just confuse people.

![image](https://user-images.githubusercontent.com/30811906/31406222-97e412c0-adce-11e7-959d-2dc8a7b3492e.png)

If not intentional, I suggest these be removed in the next version. Seems to involve many recipes concerning sulfuric acid. Could be others.

**Comment by bigbass1997** [comment]
Just going to copy paste what I said in Discord as devil's advocate.

Something to keep in mind is that we sometimes import (aka merge) changes, including recipes, from other repos into our own. So it is possible that in a different repo, such recipes did exist for some period of time, but we only recently merged them and thus only recently noticed them.

Also note that when the modpack changes to a new version, quite often GregTech and other mods will jump more than one version since the last modpack release. So in terms of the mod itself, the mod could have had recipes in it for many versions, but in terms of the modpack, maybe only a single version.

In other words the modpack versions often skip mod versions depending on their own development cycles and how long the modpack takes to release a new version.
